game/src/core/components/DrawComponent.java

```diff
@@ -344,10 +345,37 @@ public void animationMap(final Map<String, Animation> animationMap) {
    *
    * <p>Checks if the game is running in a JAR or not and will execute the corresponding loading
    * logic.
+   *
+   * <p>TODO: This is only a quick fix! We need to implement a proper way of loading assets. (See <a
+   * href="https://github.com/Dungeon-CampusMinden/Dungeon/issues/1361">Issue #1361</a>)
    */
   private void loadAnimationAssets(final IPath path) throws IOException {
+
+    Thread thread = Thread.currentThread();
```

Comment:
an dieser Methode war/bin ich schon dran

Reply:
Das ist richtig. Aber in diesem PR sehe ich langsam die nötige Reife erreicht, in dem anderen fehlt mir nach wie vor der nötige Überblick. Kann gut sein, dass ich hier morgen einen Strich drunter machen kann ....